src/qt/rpcconsole.h

```diff
@@ -58,14 +58,14 @@ class RPCConsole: public QWidget
         CMD_ERROR
     };
 
-    enum TabTypes {
-        TAB_INFO = 0,
-        TAB_CONSOLE = 1,
-        TAB_GRAPH = 2,
-        TAB_PEERS = 3
+    enum class TabTypes {
+        INFO = 0,
+        CONSOLE = 1,
+        GRAPH = 2,
+        PEERS = 3
```

Comment:
```suggestion
        PEERS,
```

No need to manually enumerate those, when the compiler does it for you